packages/core/utils/src/replaceBundleReferences.js

```diff
@@ -64,8 +64,13 @@ export function replaceURLReferences({
     }
 
     invariant(resolved.type === 'bundle_group');
-    let entryBundle = bundleGraph.getBundlesInBundleGroup(resolved.value).pop();
-    if (entryBundle.isInline) {
+    let entryAssetId = resolved.value.entryAssetId;
+    let entryBundle = bundleGraph
+      .getBundlesInBundleGroup(resolved.value)
+      .find(b => {
+        return !!b.getEntryAssets().find(a => a.id === entryAssetId);
```

Comment:
Maybe we should expose the entryAssetIds to speed this up a bit?

Reply:
Should it be `b.getMainEntry()?.id === entryAssetId` anyway? Not sure

Reply:
Not sure if it's multiple entries it might result in undefined?